utils/rpms/daos.rpmlintrc

```diff
@@ -23,6 +24,7 @@ addFilter("daos-(client|server)\.x86_64: W: dangerous-command-in-%post(un)? rm")
 addFilter("W: no-manual-page-for-binary (cart_ctl|daos_agent|dfuse|self_test|acl_dump_test|agent_tests|crt_launch|daos_debug_set_params|daos_gen_io_conf|daos_perf|daos_racer|daos_run_io_conf|daos_test|dfs_test|dfuse_test|drpc_engine_test|drpc_test|eq_tests|fault_status|hello_drpc|job_tests|jobtest|security_test|daos_firmware|daos_admin|daos_engine|daos_metrics|daos_server|daos_storage_estimator.py|evt_ctl|jump_pl_map|obj_ctl|pl_bench|rdbt|ring_pl_map|smd_ut|bio_ut|vea_stress|vea_ut|vos_perf|vos_tests|dtx_tests|ddb|ddb_tests)")
 
 addFilter("daos-(server|firmware)\.x86_64: W: non-standard-(u|g)id \/.+ daos_server")
+addFilter("daos-(server|firmware)\.x86_64: W: non-standard-(u|g)id \/.+ daos_metrics")
```

Comment:
I'm pretty sure this is in relation to the `daos_server` user, not the `daos_server` executable, so it shouldn't be needed since we don't have a `daos_metrics` user account.